src/query/ast/src/parser/expr.rs

```diff
@@ -1019,6 +1029,8 @@ pub fn expr_element(i: Input) -> IResult<WithSpan<ExprElement>> {
         |(_, not, _, _)| ExprElement::IsDistinctFrom { not: not.is_some() },
     );
 
+    let current_timestamp = value(ExprElement::CurrentTimestamp, rule! { CURRENT_TIMESTAMP });
```

Comment:
What about parsing to `ExprElement::FunctionCall` like 
```
    ExprElement::FunctionCall {
            distinct: false
            name: "current_timestamp",
            args: vec![],
            params: vec![],
            window: None,
            lambda: None,
        }
```
so that no need to add `ExprElement::CurrentTimestamp`